### PR TITLE
[TACHYON-1359] Add unit tests for GetLineageInfoListOptions

### PR DESCRIPTION
**Pull request by luoli523**
[TACHYON-1359] Add unit tests for GetLineageInfoListOptions

### CODE REVIEW

File: clients/unshaded/src/test/java/tachyon/client/lineage/options/GetLineageInfoListOptionsTest.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+public class GetLineageInfoListOptionsTest {
+
+  @Test
+  public void buildTest() {
+    new GetLineageInfoListOptions.Builder(new TachyonConf()).build();
+  }
+
+  @Test
+  public void defaultsTest() {
+    GetLineageInfoListOptions.defaults();
```
Comment (gpang) [inline clients/unshaded/src/test/java/tachyon/client/lineage/options/GetLineageInfoListOptionsTest.java]: Maybe assert that this is not null?

**Comment by gpang** [comment]
I just had a minor comment. Thanks!

**Comment by luoli523** [comment]
Thank you @gpang , I updated the PR, Please check it again.

**Comment by gpang** [comment]
LGTM

**Comment by calvinjia** [comment]
LGTM

**Pull request merged**